pygmt/src/text.py

```diff
@@ -261,9 +261,7 @@ def text_(  # noqa: PLR0912
 
     with Session() as lib:
         with lib.virtualfile_in(
-            check_kind="vector",
-            data=textfiles or data,
-            required_data=required_data,
+            check_kind="vector", data=textfiles or data, required=required_data
```

Comment:
At L193 and L194 above. The `required_data` variable wasn't changed to `required`?

Reply:
Yes, `required_data` at L193-L194 is a local variable, and `required` is not a good name for it. Perhaps rename it to `data_is_required`?